src/test/java/study/ReflectionTest.java

```diff
@@ -0,0 +1,91 @@
+package study;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import java.lang.annotation.Annotation;
+import java.lang.reflect.Constructor;
+import java.lang.reflect.Field;
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Method;
+import java.util.Arrays;
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class ReflectionTest {
+    @Test
+    @DisplayName("Car 객체 정보 가져오기")
+    void showClass() {
+        Class<Car> carClass = Car.class;
+
+        assertThat(carClass.getName()).isEqualTo("study.Car");
+    }
+
+    @Test
+    @DisplayName("test로 시작하는 메소드 실행")
+    void testMethodRun() {
+        Car car = new Car("자동차", 10000);
+        Class<Car> carClass = Car.class;
+        List<Method> testMethods = Arrays.stream(carClass.getDeclaredMethods()).filter(it -> it.getName().startsWith("test"))
+                .toList();
```

Comment:
```suggestion
        List<Method> testMethods = Arrays.stream(carClass.getDeclaredMethods())
                .filter(it -> it.getName().startsWith("test"))
                .toList();
```
가능하면 점마다 한칸 내려주시는게 가독성이 더 좋을 것 같아요 :)